src/FlipCounter.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { Component } from "react";
+import "./FlipCounter.css";
+
+class Counter extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+     x:"000000"
+    };
+  }
+  add = () => {
+    
+    this.setState({ x:"00000"+ parseInt(parseInt(this.state.x) + 1) });
+ console.log(this.state.x)
+  };
+
+  handleChange = event => {
+    this.setState({ value: event.target.value });
+    console.log(event.target.value)
+  };
+
+  handleSubmit = event => {
+    this.setState({ x: this.state.value });
+
+    event.preventDefault();
+  };
+
+  render() {
+    let mystring =this.state.x;
+   let array=["0","0","0","0","0","0"]
+   
+
+    for(var i=0;i<6;i++){   
```

Comment:
Very hard to compile this code in the head, so it is really hard to read and understand.

Let's make things easier.

You have 333 in the state as number.

1. Convert it to string.
1. Convert string to array, there is `String.split` method
2. If it's length is more than 6, need to remove first extra characters
3. It if's length is less than 6, you need to add (`6 - length`) number of zeroes in the start